src/binder/bind/copy/bind_copy_from.cpp

```diff
@@ -83,10 +83,14 @@ std::unique_ptr<BoundStatement> Binder::bindCopyNodeFrom(const Statement& statem
                 FileTypeUtils::toString(bindData->config.fileType)));
         }
     }
+    expression_vector defaultExprs;
```

Comment:
This is a fine approach but I want to suggest a conceptually more clear solution.

Consider the following example

An input CSV with 2 columns `a, b`.
A table schema with 3 columns `x, y, z`. And with default value `1, 2, 3` respectively.
And the cypher query is `COPY <table> (z, y) FROM <csv>`

============

After `bindScanSource`, we will obtain two expressions `expr(a), expr(b)` corresponding to the two columns from CSV file. 

What we are doing in this PR it to generate another vector of default expressions align with table schema, i.e. `expr(1), expr(2), expr(3)`

Then in `NodeBatchInsert`, we generate a mapping (i.e. `ColumnPositions`) to align `x` with `expr(3)`, `y` with `expr(b)` and `z` with `expr(a)`.

=============

I suggest we resolve this mapping at binder stage. So after obtaining `expr(a), expr(b)` we generate another vector expressions by reordering & padding the expressions we get from `bindScanSource`. So we expect `expr(1), expr(b), expr(a)` as the input to `NodeBatchInsert`.

This is very similar logic as what we have done in map_copy.cpp. If we do it in binder, we can remove those code in mapper. And I think we no longer need to maintain ColumnPositions in `NodeBatchInsert`